src/main/java/oss/fosslight/util/ExcelUtil.java

```diff
@@ -2858,7 +2860,195 @@ public static List<OssMaster> readOssList(MultipartHttpServletRequest req, Strin
+						HSSFRow row = sheet.getRow(rowIndex);
+						if (rowIndex == 0) {
+							//여기서 헤더 점검 필요
+							boolean validHeader = checkHeaderLicenseColumnValidate(row.cellIterator());
+							if (!validHeader){
+								log.info("The order and content of header columns must match.");
+								return null;
+							}
+						} else {
+							LicenseMaster licenseMaster = getLicenseDataByColumn(row.cellIterator());
+							if (licenseMaster != null)
+								licenseMasterList.add(licenseMaster);
+						}
+					}
+				} catch (IOException e) {
+					log.error(e.getMessage(), e);
+				} finally {
+					try {
+						wbHSSF.close();
+					} catch (Exception e) {
+					}
+				}
+			} else if ("xlsx".equals(extType) || "XLSX".equals(extType)) {
+				try {
+					wbXSSF = new XSSFWorkbook(new FileInputStream(file));
+					XSSFSheet sheet = wbXSSF.getSheetAt(0);
+					int rowSize = sheet.getPhysicalNumberOfRows();
+					for (int rowIndex = 0; rowIndex < rowSize; rowIndex++) {
+						XSSFRow row = sheet.getRow(rowIndex);
+						if (rowIndex == 0) {
+							boolean validHeader = checkHeaderColumnValidate(row.cellIterator());
+							if (!validHeader) {
+								log.info("The order and content of header columns must match.");
+								return null;
+							}
+						} else {
+							//licenseMaster 객체 생성
+							LicenseMaster licenseMaster = getLicenseDataByColumn(row.cellIterator());
+							if (licenseMaster != null)
+								licenseMasterList.add(licenseMaster);
+						}
+					}
+				} catch (IOException e) {
+					log.error(e.getMessage(), e);
+				} finally {
+					try {
+						wbXSSF.close();
+					} catch (Exception e2) {
+					}
+				}
+			}
+		}
+		//최종 그리드 리스트 생성
+		return licenseMasterList;
+	}
+
+	public static LicenseMaster getLicenseDataByColumn(Iterator<Cell> cellIterator) {
+		LicenseMaster licenseMaster = new LicenseMaster();
+		int colIndex = 0;
+		for (; cellIterator.hasNext(); colIndex++) {
+			Cell cell = (Cell) cellIterator.next();
+			String value = getCellData(cell);
+			if (colIndex == 0) {
+				if (value == null || value.trim().isEmpty()) {
+					log.debug("License Name must not be null.");
+					return null;
+				} else {
+					licenseMaster.setLicenseName(value);
+				}
+			} else if (colIndex == 1) {
+				if (value == null || value.trim().isEmpty()) continue;
+				//LICENSE TYPE 검증
+//				if (value.toLowerCase().equals("permissive")) {
+//					licenseMaster.setLicenseType(CD_LICENSE_TYPE_PMS);
+//				} else if (value.toLowerCase().equals("weak copyleft")) {
+//					licenseMaster.setLicenseType(CD_LICENSE_TYPE_WCP);
+//				} else if (value.toLowerCase().equals("copyleft")) {
+//					licenseMaster.setLicenseType(CD_LICENSE_TYPE_CP);
+//				} else if (value.toLowerCase().equals("proprietary")) {
+//					licenseMaster.setLicenseType(CD_LICENSE_TYPE_NA);
+//				} else if (value.toLowerCase().equals("proprietary free")) {
+//					licenseMaster.setLicenseType(CD_LICENSE_TYPE_PF);
+//				}

```

Comment:
If you are not using it, please delete it.

Reply:
thank you for good review.